packages/app/src/contacts/components/contacts/contacts.component.tsx

```diff
@@ -51,12 +51,12 @@ import ImportContactsFlow, {
 } from "App/contacts/components/import-contacts-flow/import-contacts-flow.component"
 import { Contact, NewContact } from "App/contacts/reducers/contacts.interface"
 import { isError } from "App/__deprecated__/common/helpers/is-error.helpers"
-import { contactsFilter } from "App/contacts/helpers/contacts-filter/contacts-filter.helper"
 import { ExportContactFailedModal } from "../export-contact-failed-modal/export-contact-failed-modal.component"
 import { applyValidationRulesToImportedContacts } from "App/contacts/helpers/apply-validation-rules-to-imported-contacts/apply-validation-rules-to-imported-contacts"
 import { ExportContactsResult } from "App/contacts/constants"
 import DeleteContactsPopup from "./delete-contacts-popup/delete-contacts-popup.component"
 import { differenceWith, isEqual } from "lodash"
+import { filterContacts } from "App/messages/helpers/messages.helpers"
```

Comment:
I'm not sure, so I'll ask if contact filtering should be the domain of messages? 🤔